test/integration/cluster/test_cluster_failover.py

```diff
@@ -0,0 +1,90 @@
+import re
+import subprocess
+
+switch_cmd_yml = r"""command: switch
+new_master: some_instance
+timeout: 30
+"""
+
+switch_cmd_timeout_yml = r"""command: switch
+new_master: some_instance
+timeout: 42
+"""
+
+
+def test_cluster_failover_switch_etcd(tt_cmd, tmpdir_with_cfg, etcd):
+    tmpdir = tmpdir_with_cfg
+
+    conn = etcd.conn()
+    switch_cmd = [tt_cmd, "cluster", "failover", "switch",
+                  f"{etcd.endpoint}/prefix", "some_instance"]
```

Comment:
Should it work if there is no `some_instance` instance in cluster?

Reply:
Yes. Return will be like:
```yml
new_master: none
command: switch
status: failed
taken_by:
  active: true
  term: 10
  uuid: de36e15a-d2a5-4015-b378-a8eb262c8ff6
  hostname: void-tt
  pid: 18209
timeout: 30
reason: Unknown instance name
```